libpq/plan.sh

```diff
@@ -0,0 +1,72 @@
+pkg_name=libpq
+pkg_origin=core
+pkg_version=9.6.3
```

Comment:
could you make it 9.6.6 please to match the pending Postgresql update PR? #980

Reply:
Will do; rerunning a local studio build with that change to make sure I got things right.